src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useContext, useState } from 'react';
+import classNames from 'classnames';
+
+import { TodoContext } from '../TodoContext';
+
+export const Notification: React.FC = () => {
+  const { errorMessage } = useContext(TodoContext);
+  const [isHidden, setIsHidden] = useState(false);
+
+  setTimeout(() => {
+    setIsHidden(true);
+  }, 3000);
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames('notification '
+            + 'is-danger '
+            + 'is-light '
+            + 'has-text-weight-normal',
```

Comment:
```suggestion
      className={classNames('notification is-danger is-light has-text-weight-normal'
       
```

Reply:
also you can add classes just splitting by comma https://www.npmjs.com/package/classnames
classNames('a', 'b', { hidden: isHidden }, 'any-class' )